### DISCUSSION TITLE
Helm value file not detecting when sync from standard repo with GitOps

### DISCUSSION THREAD

**Discussion by ysaakpr** [argoproj/argo-cd#9746] [General]
<img width="426" alt="Screenshot 2022-06-22 at 10 32 34 AM" src="https://user-images.githubusercontent.com/20730186/174947418-351500b5-1d86-4b83-8eac-7c369185befd.png">

The above i am trying to setup ambassador from ambassador repo, and in the same directory i have the values file. Which is not able to recognised while installing and Argo failed with error

<img width="715" alt="Screenshot 2022-06-22 at 10 33 49 AM" src="https://user-images.githubusercontent.com/20730186/174947563-92723315-4242-441f-b640-6ad71025b779.png">

Folder structure from git
<img width="747" alt="Screenshot 2022-06-22 at 10 35 00 AM" src="https://user-images.githubusercontent.com/20730186/174947771-addaea81-c246-469e-8259-d870f0618f98.png">

Am i doing it wrong?

**Comment by ysaakpr** [comment]
<img width="724" alt="Screenshot 2022-06-22 at 10 36 49 AM" src="https://user-images.githubusercontent.com/20730186/174947913-95a065a0-799f-4def-bea8-ce7bad95a616.png">

Followed this

**Reply by andrewmetcalfbede** [reply]
I'm having a similar issue and managed to work around it by prefixing my additional values file with `values-` i.e. `values-emissary-values.yaml`, does this work for you?

Apparently this was fixed in 2018 https://github.com/argoproj/argo-cd/issues/663 but even though I can type in the name of my file in the textbox it still doesn't work without the prefix.
